### PR TITLE
Init

### PR DESCRIPTION
**Pull request by whereasjovially**
1. Updated network params.
2. Added a new README

### CODE REVIEW

**Review by Nomad-Free-Talent — COMMENTED** [review]
Are you able to update the network_params.yaml file to only start the required services?

**Comment by whereasjovially** [comment]
> Are you able to update the network_params.yaml file to only start the required services?

It starts only tx_spammer, premetheus, grafana, dora and mev

File: README.md [hunk 1]
```diff
@@ -1,1079 +1,15 @@
...
-
-### Shadowforking
-In order to enable shadowfork capabilities, you can use the `network_params.network` flag. The expected value is the name of the network you want to shadowfork followed by `-shadowfork`. Please note that `persistent` configuration parameter has to be enabled for shadowforks to work! Current limitation on k8s is it is only working on a single node cluster. For example, to shadowfork the Holesky testnet, you can use the following command:
-```yaml
-...
-network_params:
-  network: "holesky-shadowfork"
-persistent: true
-...
+git clone https://github.com/whereasjovially/ethereum-package.git
```
Comment (Nomad-Free-Talent) [inline README.md:7]: This url should be changed for interstate-labs/ethereum-package, right?

File: README.md [hunk 2]
```diff
@@ -1,1079 +1,15 @@
...
-1. Once everything works, merge!
-
-<!------------------------ Only links below here -------------------------------->
-
-[docker-installation]: https://docs.docker.com/get-docker/
-[kurtosis-cli-installation]: https://docs.kurtosis.com/install
-[kurtosis-repo]: https://github.com/kurtosis-tech/kurtosis
-[enclave]: https://docs.kurtosis.com/advanced-concepts/enclaves/
-[package-reference]: https://docs.kurtosis.com/advanced-concepts/packages
+kurtosis run --enclave interstate-devnet github.com/whereasjovially/ethereum-package --args-file network_params.yaml
```
Comment (Nomad-Free-Talent) [inline README.md:14]: Same as above

**Review by Nomad-Free-Talent — COMMENTED** [review]
Same small issues.

**Comment by whereasjovially** [comment]
Oh, right. I missed that.

**Review by Nomad-Free-Talent — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (whereasjovially) [reply README.md]: Hello, I changed the github username to interstate-labs

**Pull request merged**